src/sentry/models/organizationmembermapping.py

```diff
@@ -21,6 +21,11 @@ class OrganizationMemberMapping(Model):
     __include_in_export__ = False
 
     organization_id = HybridCloudForeignKey("sentry.Organization", on_delete="CASCADE")
+    # TODO: allow null values for organizationmember_id column. We will later repair and populate these columns with
+    # values; and remove null=True.
+    organizationmember_id = HybridCloudForeignKey(
+        "sentry.OrganizationMember", db_index=True, null=True, on_delete="CASCADE"
+    )
```

Comment:
@markstory is this the right incantation? 

`organizationmember_id` are only unique to their region. I wasn't sure if we need them to be unique across all regions for `HybridCloudForeignKey`.